docs/docs/develop/devcontainer.md

```diff
@@ -58,6 +63,10 @@ You can either only run InvenTree or use the integrated debugger for debugging.
 
 You can now set breakpoints and vscode will automatically pause execution if that point is hit. You can see all variables available in that context and evaluate some code with the debugger console at the bottom. Use the play or step buttons to continue execution.
 
+!!! info "React Frontend development"
+
+The React frontend requires additional steps to run. Refer to
```

Comment:
please complete this sentence :)

Reply:
You beat me to it...
I was gonna make a comment that the plan is to add a link to the new doc, haha